src/components/others/formCreate/index.jsx

```diff
@@ -118,11 +106,34 @@ export default function FormCreate({ handleSave, controlsState, updateControlsSt
           onBlur={() => handleBlur("description")}
           value={controlsState.description}
         />
-
         {(!focusedFields.description || controlsState.description !== "") && (
           <ErrorDisplay message={errors.description} />
         )}
       </div>
+      <fieldset className={styles.textFormat}>
+        <input
+          type="radio"
```

Comment:
radio buttons are missing design stying. default HTML styling appled.

Reply:
Yeah, on purpose. The design is not ready yet.